api/src/pcapi/routes/backoffice/templates/components/forms/string_field.html

```diff
@@ -1,6 +1,7 @@
-<div class="form-floating {% if field.type == "PCIntegerField" or field.type == "PCOptIntegerField" %} d-inline-block {% elif field.type == "PCOptHiddenField" or field.type == "PCPostalCodeHiddenField" or field.type == "PCOptPostalCodeHiddenField" or field.type == "PCOptHiddenIntegerField" or field.type == "PCHiddenField" %} d-none {% endif %} pc-field-{{ field.name }}">
+<div class="form-floating {% if "Hidden" in field.type %} d-none {% elif "Integer" in field.type or "Decimal" in field.type %} d-inline-block {% endif %} pc-field-{{ field.name }}">
```

Comment:
Je n'aime pas trop le `in` dans le nom, car ça peut faire entrer dans le critère de nouveaux champs qu'on n'a pas prévu de faire tomber dans ces `if`. Une propriété sur la classe me paraîtrait plus propre, du genre : 

```
class HiddenField:
    input_type = "hidden"

class NumberField:
    input_type = "number"

class PCPostalCodeHiddenField(PCPostalCodeField, HiddenField):
    ...

class PCIntegerField(wtforms.IntegerField, NumberField):
    ...
```
```
{% if field.is_hidden %} d-none {% elif ...
```
```
<input type="{{ field.input_type or "text" }}" ... >
```